### ISSUE TITLE
Copying a canvas to an empty canvas causes an error

### ISSUE THREAD

**Issue by patchuby** [tabularelf/Canvas#9]
It will throw the following error : 
```Canvas Error: Canvas has no data or in use!
 at gml_Script___CanvasError (line 3) -        show_error("Canvas Error: " + _str, true);```

**Comment by tabularelf** [comment]
I wasn't able to look at the full issue at the time as I was at work, but for the record, the full error message is more preferable.
At least I can remember what this is about haha

**Comment by tabularelf** [comment]
Implemented under ecabef9364df34ae10f4be7bcc49212fcfa3747b.
